Makefile

```diff
@@ -31,6 +31,12 @@ build:
 	@go build -o build/ ${LDFLAGS} ./cmd/celestia
 .PHONY: build
 
+## build-jemalloc: Build celestia-node binary with jemalloc allocator for BadgerDB instead of Go's native one
```

Comment:
it'd be good to add a little description here of why you might want to use build-jemmaloc and the context, for testing only? or is it possible that you as a node runner might choose to run this way and are at your own risk etc

Reply:
I wanted to do it on the docs side, but we can also do it here